operations/http.go

```diff
@@ -605,6 +606,31 @@ func (ac *legacyClient) PutPatch(incomingPatch patchSubmission) (*patch.Patch, e
 	return reply.Patch, nil
 }
 
+func (ac *legacyClient) GetRawPatchWithModules(patchId string) (data.RawPatch, error) {
```

Comment:
nit: should this return a `*data.RawPatch`? That way, you can return nil in case of error.